core/build.gradle

```diff
@@ -21,8 +21,11 @@ archivesBaseName = "core"
 
 ext {
     bouncyCastleVersion = "1.75"
+
     caffeineVersion = "3.1.7"
+
     zstdVersion = "1.5.5-5"
+
     jacksonVersion = "2.15.2"
```

Comment:
Can we have a comment about it(since we can not enforce) to remember that there should be a new line after each version? And could you please move the rest of the versions to the variables to have the same style?